### ISSUE TITLE
Read from file as default

### ISSUE THREAD

**Issue by Splines** [Jonathan-LeRoux/IguanaTex#66]
Hey Jonathan, thanks a lot for this amazing PowerPoint plugin, exactly what I was looking for and works like a charm!  ❤

For convenience, it'd be amazing if the `Read form file` path for inserting a new equation or editing an existing one, could be stored as default value. This way, I wouldn't have to copy and paste the file path all the time.

**Comment by Jonathan-LeRoux** [comment]
Thanks for the suggestion. Are you referring to the path including the file name or just the folder? Would it be okay to default to the last used folder?

**Comment by Splines** [comment]
If the path defaulted to the last used _folder_, I would still have to append my filename after it, right? In this case, I'd love the path to also include the _file name_ (e.g. `tmp.tex`) such that the only thing I really have to do is `Insert new equation` -> `Load File` -> `Generate LaTeX` (and maybe even `Load file` could be omitted).

**Comment by Jonathan-LeRoux** [comment]
As mentioned in issue https://github.com/Jonathan-LeRoux/IguanaTex/issues/68, I implemented this in this beta version: 
[IguanaTex_v1_61_1_20240726.zip](https://github.com/user-attachments/files/16397753/IguanaTex_v1_61_1_20240726.zip).
You need to click "Make Default" when on the "Read from file" page. Actually, in this new version, "Make Default" only saves the settings that are currently active (e.g., it won't save the DPI if Shape is selected).
Please give it a try.

**Comment by Splines** [comment]
Thank you so much. The new button is great and it works as expected. `Make default` also works.

Note that when I enter a wrong file path, the `Load file` button is grayed out, while the `Load File & Generate` button is not. Clicking it with an invalid file path specified, this results in the following error:


```
Run-time error '3001':

Arguments are of the wrong type, are out of acceptable range, or are in conflict with one another.
```

**Comment by Jonathan-LeRoux** [comment]
Good catch, thanks! It's will be fixed in the final version.

**Comment by Jonathan-LeRoux** [comment]
Fixed with [v1.62 release](https://github.com/Jonathan-LeRoux/IguanaTex/releases/tag/v1.62).

**Comment by Splines** [comment]
Thanks a lot for addressing this, much appreciated, this really improves the workflow.
